rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/LanguageContributionsMultiplexer.java

```diff
@@ -159,6 +161,8 @@ private synchronized void calculateRouting() {
         hasReferences = anyTrue(ILanguageContributions::hasReferences);
         hasImplementation = anyTrue(ILanguageContributions::hasImplementation);
 
+        specialCaseHighlighting = anyTrue(ILanguageContributions::specialCaseHighlighting);
```

Comment:
for the parser we do something special, we say `firstOrFail` can we do something similar here as well? so that we only pickup the parsing status from the first contribution?